deployment/GP03FLMA_Deploy.csv

```diff
@@ -143,39 +143,39 @@ SKQ202111S,,SKQ202208S,,GP03FLMA-RIS01-00-SIOENG000,8,1,2021-07-23T19:43:00,2022
+SKQ202208S,,SKQ202308S,,GP03FLMA-RIS01-03-DOSTAD000,9,1,2022-05-19T17:27:00,2023-05-21T16:44:00,CGMGP-03FLMA-00009,,CGINS-DOSTAD-00439,50.021870,-144.359633,,30,4109,,,
+SKQ202208S,,SKQ202308S,,GP03FLMA-RIS01-04-PHSENF000,9,1,2022-05-19T17:27:00,2023-05-21T16:44:00,CGMGP-03FLMA-00009,,CGINS-PHSENF-P0102,50.021870,-144.359633,,30,4109,,,
+SKQ202208S,,SKQ202308S,,GP03FLMA-RIS01-05-FLORTD000,9,1,2022-05-19T17:27:00,2023-05-21T16:44:00,CGMGP-03FLMA-00009,,CGINS-FLORTD-00993,50.021870,-144.359633,,30,4109,,,
+SKQ202308S,,,,GP03FLMA-RIM01-00-SIOENG000,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGCON-SIOCON-14871,49.978,-144.246,,1507,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-ADCPSL003,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-ADCPSL-22846,49.978,-144.246,,500,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOG040,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOG-11594,49.978,-144.246,,30,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOG041,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOG-13600,49.978,-144.246,,40,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOG042,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOG-11638,49.978,-144.246,,60,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOG043,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOG-11604,49.978,-144.246,,90,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOG044,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOG-11534,49.978,-144.246,,130,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOG045,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOG-11645,49.978,-144.246,,180,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOG046,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOG-13604,49.978,-144.246,,250,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOG047,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOG-11611,49.978,-144.246,,350,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOG048,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOG-10253,49.978,-144.246,,500,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOH049,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOH-12227,49.978,-144.246,,750,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOH050,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOH-12619,49.978,-144.246,,1000,4127,,,
+SKQ202308S,,,,GP03FLMA-RIM01-02-CTDMOH051,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-CTDMOH-11703,49.978,-144.246,,1500,4127,,,
+SKQ202308S,,,,GP03FLMA-RIS01-00-SIOENG000,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGCON-SIOCON-13997,49.978,-144.246,,30,4127,,,
+SKQ202308S,,,,GP03FLMA-RIS01-03-DOSTAD000,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-DOSTAD-00503,49.978,-144.246,,30,4127,,,
+SKQ202308S,,,,GP03FLMA-RIS01-04-PHSENF000,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-PHSENF-P0080,49.978,-144.246,,30,4127,,,
+SKQ202308S,,,,GP03FLMA-RIS01-05-FLORTD000,10,1,2023-05-18T19:23:00,,CGMGP-03FLMA-00010,,CGINS-FLORTD-01116,49.978,-144.246,,30,4127,,,
```

Comment:
Looks good. I looked at previous deployments for expectations for recovery at anchor vs midwater.

Reply:
checked. looks good